libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/FlyoutObjectDetection.tsx

```diff
@@ -100,15 +81,17 @@ export class FlyoutObjectDetection extends React.Component<
       });
     }
   }
+
   public render(): React.ReactNode {
     const { isOpen } = this.props;
-    const item = this.state.item;
+    const item = this.props.item; // state.item is undefined during 1st execution
```

Comment:
https://stackoverflow.com/questions/27991366/what-is-the-difference-between-state-and-props-in-react

Makes sense that you would want props for the 1st execution, but what happens when state.item is updated? Looks like props is immutable, so we might be missing changes to the item during a state change, right?